lib/puppet/face/ca.rb

```diff
@@ -50,9 +50,9 @@
     end
 
     when_invoked do |options|
-      raise "Not a CA" unless Puppet::SSL::CertificateAuthority.ca?
+      raise _("Not a CA") unless Puppet::SSL::CertificateAuthority.ca?
```

Comment:
Thoughts on translator notes to ignore CA here and below? Maybe not necessary since it's an abbreviation?